aten/src/ATen/cuda/CUDABlas.cpp

```diff
@@ -1341,13 +1364,29 @@ void int8_gemm(
       mat2_ptr,
       Bdesc.descriptor(),
       &beta_val,
+#ifdef USE_ROCM
       result_ptr,
+#else
+      nullptr,
```

Comment:
I don't think we want to change existing behavior when USE_ROCM is not defined? Now we pass nullptr instead of result_ptr?